ts/src/facilityCore.ts

```diff
@@ -17,7 +17,7 @@ export interface IServiceError {
 	/** The error message. (For developers, not end users.) */
 	message?: string;
 	/** Advanced error details. */
-	details?: any;
+	details?: Record<string, unknown>;
```

Comment:
This could be a breaking change. I changed it because eslint's default rules complain about `any`. The alternative is to ignore that rule, either on this line or project-wide.

Reply:
I have no objection to this change.